vms/volunteer/views.py

```diff
@@ -149,8 +149,8 @@ class ShowReportListView(LoginRequiredMixin, ListView):
 
     def post(self, request, *args, **kwargs):
         volunteer_id = self.kwargs['volunteer_id']
-        event_list = get_signed_up_events_for_volunteer(volunteer_id)
-        job_list = get_signed_up_jobs_for_volunteer(volunteer_id)
+        event_list = get_events_ordered_by_name()
+        job_list = get_jobs_ordered_by_title()

```

Comment:
I will take a look again. I think I had some misunderstanding with the code base. I think I should close this pull request.